Resources/Prototypes/Entities/Mobs/Species/diona.yml

```diff
@@ -110,9 +110,12 @@
           32:
             sprite: Mobs/Species/Human/displacement.rsi
             state: jumpsuit-female
-  - type: FootPrints # DeltaV port from EE, blood splatter
+  - type: FootPrints # begin DeltaV
```

Comment:
should really just be two separate comments, ond on the footprints and one on the movement modifier comp